docs/DeveloperGuide.md

```diff
@@ -150,37 +150,36 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
-|* * *   |Passenger                  |delete my profile                                                |my data will not be stored when I have stopped the service                                      |
-|* *     |Driver                     |filter passengers' destination and pickup point based on location|I don't have to spend too long to pick up or drop off passengers and minimise my travelling time|
-|* *     |Female Passenger           |find only female drivers                                         |I can be comfortable                                                                            |
-|*       |User concerned with privacy|limit the information I want to disclose                         |my private information exposed is limited                                                       |
-|*       |Driver                     |Indicate one-off trips                                           |my passengers would understand this is not a recurring trip                                                                                                |
-|* *     |Passenger                  |edit drop off location                                           |change the destination if needed and be more flexible                                           |
-|*       |Passenger                  |indicate the price willing to pay                                |I can incentivise the drivers to more likely to choose me and pick me up on time                |
+|Priority|As a …​    |I want to …​                                                  |So that                                                                                         |
+|--------|--------------|-----------------------------------------------------------------|------------------------------------------------------------------------------------------------|
+|* * *   |HR Executive  |allocate drivers to passengers to be picked up                   |I can arrange carpooling trips for my colleagues                                                |
+|* * *   |HR Executive  |search for specific type of passengers                           |I can see if any passengers fulfil a criteria and view their carpool details                    |
+|* * *   |HR Executive  |list all passengers                                              |I can see all the passengers available                                                          |
+|* * *   |HR Executive  |create a profile                                                 |I can easily manage and track drivers and passengers                                            |
+|* * *   |HR Executive  |delete my profile                                                |passenger's data will not be stored when they are no longer looking to carpool                  |
```

Comment:
I think you might have to 'change delete my profile to 'delete employee profile' as the user is the HR exec now and they're managing other employee's profiles.

Reply:
Good catch, let me amend that